sklearn/utils/testing.py

```diff
@@ -331,11 +340,7 @@ class _IgnoreWarnings(object):
 
```

Comment:
The above docstring will have to be changed to reflect the fact that it can now accept a category kwarg in its constructor.